buildSrc/src/main/minimumRuntime/org/elasticsearch/gradle/VersionProperties.java

```diff
@@ -18,17 +25,41 @@ public static String getLucene() {
         return lucene;
     }
 
+    /**
+     * Return the bundled jdk version, broken into elements as:
+     * [feature, interim, update, build]
+     *
+     * Note the "patch" version is not yet handled here, as it has not yet
+     * been used by java.
+     */
+    public static List<String> getBundledJdk() {
```

Comment:
Too bad that Gradles `JavaVersion` doesn't have all this information.
Given how broadly this  class could be used it might be better to wrap the list in a class, or move the parsing logic to the build script that uses it ?

Reply:
I don't think it should be used broadly? I'm fine moving it to distribution/build.gradle if you insist, but I think it makes sense here since VersionProperties is all about the versions.properties file where this setting lives.

Reply:
I'm ok to return it as a string from here and do the parsing in `distribution/build.gradle` since it's only needed there.